### PR TITLE
Host release prep PR v4.26.0

### PR DESCRIPTION
**Pull request by v-smanchem**
### Issue describing the changes in this PR

Bumping up the minor version and updated the release-notes.

* [x] My changes **do not** require documentation changes
    * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **should not** be added to the release notes for the next release
    * [x] Otherwise: I've added my notes to `release_notes.md`
* [x] My changes **do not** need to be backported to a previous version
    * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [x] My changes **do not** require diagnostic events changes
    * Otherwise: I have added/updated all related diagnostic events and their documentation (Documentation issue linked to PR)
* [x] I have added all required tests (Unit tests, E2E tests)

### CODE REVIEW

File: release_notes.md [hunk 1]
```diff
@@ -4,19 +4,7 @@
...
-- Update Node.js Worker Version to [3.8.0](https://github.com/Azure/azure-functions-nodejs-worker/releases/tag/v3.8.0)
-- Identity dependencies updated to 6.31.0:
-  - Microsoft.IdentityModel.Tokens
-  - System.IdentityModel.Tokens.Jwt
-  - Microsoft.IdentityModel.Abstractions
-  - Microsoft.IdentityModel.JsonWebTokens
-  - Microsoft.IdentityModel.Logging
-- Updated Grpc.AspNetCore package to 2.55.0 (https://github.com/Azure/azure-functions-host/pull/9373)
-- Update protobuf file to v1.10.0 (https://github.com/Azure/azure-functions-host/pull/9405)
 - Send an empty RpcHttp payload if proxying the http request to the worker (https://github.com/Azure/azure-functions-host/pull/9415)
```
Comment (VpOfEngineering) [inline release_notes.md]: I see lines 20-22 in the previous version's release notes.
https://github.com/Azure/azure-functions-host/blob/release/4.x/release_notes.md
Is there any reason they are still included here?

**Review by VpOfEngineering — CHANGES_REQUESTED** [review]

File: release_notes.md (hunk 1 above)
Comment (v-smanchem) [reply release_notes.md]: Updated the release notes

File: build/common.props [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <TargetFramework>netstandard2.1</TargetFramework>
     <LangVersion>latest</LangVersion>
     <MajorVersion>4</MajorVersion>
-    <MinorVersion>25</MinorVersion>
+    <MinorVersion>26</MinorVersion>
```
Comment (VpOfEngineering) [inline build/common.props:7]: Please change the PatchVersion following [SemVer](https://semver.org/). Thanks!

**Review by VpOfEngineering — CHANGES_REQUESTED** [review]

File: build/common.props (hunk 2 above)
Comment (VpOfEngineering) [reply build/common.props:7]: WOW wrong PR haha.

**Review by liliankasem — APPROVED** [review]

File: release_notes.md [hunk 3]
```diff
@@ -4,20 +4,4 @@
 - My change description (#PR)
 -->
 - Increase timeout for http proxying requests [#9433](https://github.com/Azure/azure-functions-host/pull/9433)
-
 - Update Python Worker Version to [4.14.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.14.0)
```
Comment (VpOfEngineering) [inline release_notes.md]: Please compare the file that is currently in dev to the latest tag (4.25.2) and remove the updates that are in 4.25.2. Thank you!

**Review by VpOfEngineering — CHANGES_REQUESTED** [review]

**Pull request closed without merging**